grammars/css.cson

```diff
@@ -483,6 +483,10 @@
         'include': '#comment-block'
       }
       {
+        'match': '--([A-Za-z0-9_-]+)'
```

Comment:
"------" is indeed a really lousy name, but it's valid (test it yourself), so I think we should allow it (to be tokenized) and let people make their own decisions.